src/exec_from.cr

```diff
@@ -1,4 +1,12 @@
 module ExecFrom
+  struct Result
+    getter status : Process::Status
+    getter output : IO
+
+    def initialize(@status, @output)
+    end
+  end
```

Comment:
```suggestion
  record Result, status : Process::Status, output : IO
```

Reply:
I don't like that records are mutable, any objection to keeping this immutable?

Reply:
`record` is just a macro that spits out an [immutable `Struct` by default](https://github.com/crystal-lang/crystal/blob/dd40a2442fa186add8a82b74edb14a90aa1dae05/src/macros.cr#L62-L69).

Reply:
ohhhh great, i thought it had created `property`s. excellent.